features/distributed/dao-test/src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -0,0 +1,23 @@
+<blueprint xmlns="http://www.osgi.org/xmlns/blueprint/v1.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+           xmlns:cm="http://aries.apache.org/blueprint/xmlns/blueprint-cm/v1.3.0" xmlns:ext="http://aries.apache.org/blueprint/xmlns/blueprint-ext/v1.5.0"
+           xsi:schemaLocation="
+        http://www.osgi.org/xmlns/blueprint/v1.0.0
+        https://osgi.org/xmlns/blueprint/v1.0.0/blueprint.xsd
+        http://aries.apache.org/blueprint/xmlns/blueprint-cm/v1.3.0
+        http://aries.apache.org/schemas/blueprint-cm/blueprint-cm-1.3.0.xsd
+        http://aries.apache.org/blueprint/xmlns/blueprint-ext/v1.5.0
+        http://aries.apache.org/schemas/blueprint-ext/blueprint-ext-1.5.xsd
+">
+
+    <!-- The Spring Application Context exposing the DAOs is first loaded AFTER the dao-impl bundle has
+    already been successfully started. Therefore it could be possible that the DaoServiceTester already tries to
+    dynamically pull in services, which have not yet been exposed. Therefore we reference at least one DAO here, to
+    ensure that gemini blueprint is doing its magic -->
```

Comment:
Make this more clear. What it means is: Wait for gemini blueprint to expose all services before dao-test tries to dynamically pull it in via `bundleContext.getServiceReference(clazz)`;